sphinx/source/docs/releases/0.12.4.rst

```diff
@@ -125,6 +125,12 @@ BokehJS has migrated to using ``.tsx`` templates. Provisional support for
 including ``.eco`` templates has been removed from both BokehJS and the
 Bokeh Python library.
 
+Used base64 serialisation
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Changes made to use b64encode mean that plotting numpy/pandas data no longer
+works with Python 3.3.
```

Comment:
A workaround should be to convert the data to pure python lists, which is also probably worth mentioning.